code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -347,12 +364,13 @@
 		C.soundbang_act(1, 100, rand(0, 5))
 
 /datum/chemical_reaction/phlogiston
-	name = /datum/reagent/phlogiston
+	name = "phlogiston"
```

Comment:
capitalize Phlogiston ill go mad

Reply:
None of the reaction names are capitalized. Doing so would result in randomly capitalized words in the middle of a statement.